# HG changeset patch
# User pm
# Date 1577355379 -3600
#      Thu Dec 26 11:16:19 2019 +0100
# Node ID 6fa079d1c99cbdd7670e877a1df95589900038c7
# Parent  759d9992c6c013739b3e9e04aa56a117d7d4289e
Ticket mantis 0005502: Options de discussion en trop dans le menu contextuel d'un message

diff --git a/comm-esr60/comm/mail/base/content/mailWindowOverlay.xul b/comm-esr60/comm/mail/base/content/mailWindowOverlay.xul
--- a/comm-esr60/comm/mail/base/content/mailWindowOverlay.xul
+++ b/comm-esr60/comm/mail/base/content/mailWindowOverlay.xul
@@ -927,8 +927,8 @@
                   command="cmd_watchThread"/>
       </menupopup>
     </menu>
-    
     <menuseparator id="paneContext-afterMove"/>
+    <!--
     <menuitem id="mailContext-ignoreThread"
               label="&contextKillThreadMenu.label;"
               accesskey="&contextKillThreadMenu.accesskey;"
@@ -941,7 +941,7 @@
               label="&contextWatchThreadMenu.label;"
               accesskey="&contextWatchThreadMenu.accesskey;"
               command="cmd_watchThread"/>
-    <!--
+
     <menuseparator id="mailContext-afterWatchThread"/>
     -->
     <menuitem id="mailContext-saveAs"
